lib/backburner.js

```diff
@@ -73,6 +79,48 @@ Backburner.prototype = {
     }
   },
 
+  _trigger: function(eventName /* ...args */) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      var len = arguments.length;
```

Comment:
then lets hard code it to exactly 2, and document it as such. No need then in touching arguments. at all.